### DISCUSSION TITLE
filterTaps with preventDefault disables all clicks?

### DISCUSSION THREAD

**Discussion by Linksku** [pmndrs/use-gesture#524] [Q&A]
**Describe the bug**
I'm using use-gesture to handle the back swipe gesture myself, so in browsers with a builtin back swipe gesture, it'll go back twice. I wanted to use `preventDefault` to disable the builtin gestures. However, if I enable `preventDefault`, it seems to disable all clicks. I assumed `filterTaps` would ignore all taps/clicks.

I tried calling handling the `touchstart` event myself and calling `preventDefault` (as recommended on Stackoverflow for disabling native swipe), but this seems to break use-gesture.

Is there an easy way to disable native gestures and use use-gesture?

**Information:**
 - Use Gesture version: v10.2.16
 - Device: iPhone 11/Thinkpad
 - OS: iOS 15/Win10
 - Browser: Safari/Chrome

**Checklist:**
  - [x] I've read the [documentation](https://use-gesture.netlify.app/).
  - [x] If this is an issue with drag, I've tried setting `touch-action: none` to the draggable element.

**Comment by dbismut** [comment]
Did you read the documentation on `filterTaps`? `filterTaps` `preventDefault` clicks that happen after a drag (ie a movement greater than 3px).

Not sure I have a good answer to your problem. Is there a way, without useGesture to disable the built-in back gesture?
